pkg/model/bootstrapscript.go

```diff
@@ -247,7 +247,7 @@ func (b *BootstrapScript) buildEnvironmentVariables(cluster *kops.Cluster) (map[
 // ResourceNodeUp generates and returns a nodeup (bootstrap) script from a
 // template file, substituting in specific env vars & cluster spec configuration
 func (b *BootstrapScriptBuilder) ResourceNodeUp(c *fi.CloudupModelBuilderContext, ig *kops.InstanceGroup) (fi.Resource, error) {
-	keypairs := []string{"kubernetes-ca", "etcd-clients-ca"}
+	keypairs := []string{"kubernetes-ca", "etcd-clients-ca", "kops-controller-bootstrap"}
```

Comment:
I would call it just `bootstrap`, but whatever.

Not all nodes need this. It should only be added to keypairs if cloudprovider is openstack and instanceGroup is control plane.

Reply:
Correction: only add this to keypairs if Openstack.

And the name should end in `-ca` in order for the keypair to be rotatable. So `bootstrap-ca`. Otherwise we need to modify `rotatableKeysetFilter()`.